app/split_shares.go

```diff
@@ -113,8 +113,8 @@ func SplitShares(txConf client.TxConfig, squareSize uint64, data *core.Data) ([]
 // that message and their corresponding txs get written to the square
 // atomically.
 type shareSplitter struct {
-	txWriter  *coretypes.ContiguousShareWriter
-	msgWriter *coretypes.MessageShareWriter
+	txWriter  *shares.ContiguousShareSplitter
+	msgWriter *shares.MessageShareSplitter
```

Comment:
quite a bit of code had to be changed in the shareSplitter in order to update it to work with the local copies of the share encoding code, as that was never done before as I assumed we'd merge non-interactive defaults code first :sweat_smile: 

would prefer not create a separate PR for this, but I understand if its something we have to do